pkg/flowkit/account.go

```diff
@@ -41,6 +41,10 @@ type Account struct {
 	//
 	// Ref: https://docs.onflow.org/flow-cli/security/#private-account-configuration-file
 	fromFile string
+	// saveAdvancedFormat is to save an account in advanced format even if it has default values as defined
+	// in isDefaultKeyFormat() in flowkit/config/json/account.go
+	// defaults to false
+	saveAdvancedFormat bool
```

Comment:
```suggestion
	// useAdvancedSaveFormat can be used to indicate that this account should be saved in advanced format
	//  even if it uses default values (as defined in isDefaultKeyFormat() in flowkit/config/json/account.go).
	// 
	// This field defaults to false.
	useAdvancedSaveFormat bool
```

`useAdvancedSaveFormat` reads a bit more clearly to me. Great way to implement the advanced saving!

Reply:
addressed